dataservice/api/phenotype/schemas.py

```diff
@@ -0,0 +1,29 @@
+from marshmallow_sqlalchemy import field_for
+from marshmallow import validates, ValidationError
+
+from dataservice.api.phenotype.models import Phenotype
+from dataservice.api.common.schemas import BaseSchema
+from dataservice.api.common.validation import validate_age
+from dataservice.extensions import ma
+
+
+class PhenotypeSchema(BaseSchema):
+    # Should not have to do this, since participant_id is part of the
```

Comment:
We should consider getting rid of this comment. If we want to do something about the problem, we should find a different way to track it.